### PR TITLE
Update supported metadata list

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Update list of supported metadata for performance enhancements

### CODE REVIEW

File: docs/_articles/ja/user-guide/perf-enhancements.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ lang: ja
 - Visualforce コンポーネント
 - Visualforce ページ
 - Lightning コンポーネント
```
Comment (xyc) [inline docs/_articles/ja/user-guide/perf-enhancements.md:23]: Wait for translation here?

File: docs/_articles/en/user-guide/perf-enhancements.md [hunk 2]
```diff
@@ -20,4 +20,5 @@ In this beta release, performance enhancements are effective when you run the **
 - Apex Trigger
 - Visualforce Component
 - Visualforce Page
-- Lightning Components
+- Lightning Web Components
```
Comment (hbc-bindu) [inline docs/_articles/en/user-guide/perf-enhancements.md:23]: @lcampos, now that performance enhancements cover all components (Apex, Visualforce, LWC and Aura), should we delete these lines from intro para:

Currently, these changes only support Apex and Visualforce metadata types. We’ll add support for more metadata types in future releases.

File: docs/_articles/ja/user-guide/perf-enhancements.md (hunk 1 above)
Comment (sfsholden) [reply docs/_articles/ja/user-guide/perf-enhancements.md:23]: Translation could take awhile. It's probably best that we still add it in English.

File: docs/_articles/en/user-guide/perf-enhancements.md (hunk 2 above)
Comment (lcampos) [reply docs/_articles/en/user-guide/perf-enhancements.md:23]: Good catch, let me update it to something more generic and still keep that the coverage is limited but we'll be expanding it as time goes by.

File: docs/_articles/ja/user-guide/perf-enhancements.md (hunk 1 above)
Comment (xyc) [reply docs/_articles/ja/user-guide/perf-enhancements.md:23]: there could be some inconsistency - Lightning コンポーネント translates to "Lightning component" on Google

Comment (xyc) [reply docs/_articles/ja/user-guide/perf-enhancements.md:23]: Google says "Lightning web component" is "Lightning Webコンポーネント"

**Pull request merged**